script/eslint-plugin-va/rules/use-resolved-path.js

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable va/use-resolved-path */
+// self rule is disabled because it contains the string needed for comparison
+const MESSAGE = 'Use resolved path and remove unnecessary parent path';
+
+function isIncluded(val, path) {
+  const isString = str => typeof str === 'string';
+  if (!isString(val) || path === null) {
+    return false;
+  }
+  return val.includes(path);
+}
```

Comment:
I'm wondering if there is part of the AST that is an "import" node.  If it's possible to get that, we should be able to check if the file specifically _imports_ the string instead of checking to see if a string is included anywhere in the file.